nvm.sh

```diff
@@ -130,7 +130,7 @@ nvm_has_system_iojs() {
 }
 
 nvm_is_version_installed() {
-  [ -n "${1-}" ] && [ -d "$(nvm_version_path "${1-}" 2> /dev/null)" ]
```

Comment:
We'll need to keep the `[ -n "${1-}" ]` test, since `nvm_version_path` will error on empty input.

Reply:
That check was actually unnecessary before as without it, `nvm_version_path` will print "version is required" to stderr which is eaten by `/dev/null`, and so `[ -d "" ]` will always exit with 1 as expected.

But with the new change, the check is actually required since we don't want the test argument to evaluate to `/bin/node` if `$1` isn't present. Good catch!